src/libostree/ostree-sysroot.h

```diff
@@ -32,6 +32,16 @@ G_BEGIN_DECLS
 #define OSTREE_IS_SYSROOT(obj) \
   (G_TYPE_CHECK_INSTANCE_TYPE ((obj), OSTREE_TYPE_SYSROOT))
 
+/**
+ * OSTREE_SYSROOT_LIVE_PATH:
+ *
+ * The existence of this file declares that the system
+ * is booted "live" (fully in RAM).
```

Comment:
I think one important thing worth noting here is that unlike the seemingly similar `/run/ostree-booted`, this isn't written by OSTree itself. I.e. it's an agreed-upon API between OSTree and OS vendors. Maybe something like:

> It is written by external code which drive OSTree in live contexts (e.g. live ISOs or live PXE boots).

?